build.wake

```diff
@@ -187,35 +187,41 @@ def tarball Unit =
     def wakeSourcesString =
         wakePaths
         | map (_.getPathName.format)
-        | foldr (_, ",\n  ", _) Nil
+        | foldr (_, ",\n    ", _) Nil
```

Comment:
Why is this space being added?

Reply:
See the other comment; tl;dr it ensures `manifest.wake` is formatted

Reply:
Ah ok gotcha